src/libraries/Microsoft.PowerFx.Json/FormulaValueJSON.cs

```diff
@@ -32,10 +31,22 @@ public static FormulaValue FromJson(string jsonString, FormulaType formulaType =
       
         public static FormulaValue FromJson(string jsonString, FormulaValueJsonSerializerSettings settings, FormulaType formulaType = null)
         {
-            using JsonDocument document = JsonDocument.Parse(jsonString);
-            JsonElement propBag = document.RootElement;
-
-            return FromJson(propBag, settings, formulaType);
+            try
+            {
+                using JsonDocument document = JsonDocument.Parse(jsonString);
+                JsonElement propBag = document.RootElement;
+
+                return FromJson(propBag, settings, formulaType);

```

Comment:
>FromJson [](http://example.com/codeflow?start=23&length=8)

From caller's perspective, there are 4 overloads of FromJson(...) ... which ones catch? 

They should probably all behave consistently (all catch or none catch).

Reply:
Only the one that uses a string
Others use JsonElement

Reply:
per-offline - once we have a JsonElement, we don't get parser errors.